ports/servoshell/desktop/app.rs

```diff
@@ -434,11 +454,68 @@ impl App {
                         webview.go_forward(1);
                     }
                 },
-                WebDriverCommandMsg::SendKeys(..) |
-                WebDriverCommandMsg::KeyboardAction(..) |
-                WebDriverCommandMsg::MouseButtonAction(..) |
-                WebDriverCommandMsg::MouseMoveAction(..) |
-                WebDriverCommandMsg::WheelScrollAction(..) |
+                // Key events don't need hit test so can be forwarded to constellation for now
+                WebDriverCommandMsg::SendKeys(..) => {
+                    running_state.forward_webdriver_command(msg);
+                },
+                WebDriverCommandMsg::KeyboardAction(..) => {
+                    running_state.forward_webdriver_command(msg);
+                },
```

Comment:
Eventually these should use `notify_input_event`. That would be good to do in a followup.